sycl/test-e2e/DeviceArchitecture/device_architecture_comparison_on_device_aot.cpp

```diff
@@ -1,7 +1,8 @@
 // REQUIRES: arch-intel_gpu_pvc, ocloc
 
-// XFAIL: arch-intel_gpu_pvc && opencl && igc-dev
-// XFAIL-TRACKER: https://github.com/intel/llvm/issues/16401
+// Test flakily fails on PVC + OpenCL
+// UNSUPPORTED: arch-intel_gpu_pvc && opencl
+// UNSUPPORTED-TRACKER: https://github.com/intel/llvm/issues/16662
```

Comment:
I suggest we immediately fix the test instead: I believe that `.wait()` is missing on the actual kernel which fills `result` and since `result` is not a buffer which blocks on destruction to wait for all accessors using it we sometimes use it too early, i.e. before kernel has even completed/started.